notebooks/data_quality.ipynb

```diff
@@ -0,0 +1,1833 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">DL1_datacheck_2023*
Also, at least for me, the ? symbol does not work as a wildcard for globbing as if it were the shell
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Doesn't it? (and if you just try in a python session?)
As for the specific path in the comments in the notebook, there is an error of mine (asked for 2022 directories and then for 2023 files). Will fix.

Reply:
> Doesn't it? (and if you just try in a python session?) It works for me.

Sorry, yes, it works in both notebooks and python interactive sessions. My bad